scipy/io/mmio.py

```diff
@@ -30,7 +33,7 @@ def mminfo(source):
     Parameters
     ----------
 
-    source : file
+    filename : file
```

Comment:
I think `source` is fine, and better than `filename`, as the parameter value is not always a file name.

I see the parameter is `target` when it is a write operation, that doesn't seem too bad either.  My vote would be to revert to the old names.